### PR TITLE
New module to restore State to Project Stages

### PR DESCRIPTION
**Pull request by dreispt**
The State can be useful for action rules or record rules that operate depending on some "canonical" states.
See the README.md.

### CODE REVIEW

File: project_stage_state/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+import project
```
Comment (pedrobaeza) [inline project_stage_state/__init__.py]: `from . import project`

**Comment by pedrobaeza** [comment]
Anyway, I think it's better to link business logic to 1..n stages instead of making this double link. Let's see what people think.

**Comment by dreispt** [comment]
Fixed.

**Comment by dreispt** [comment]
I intend to use this for teh `project_baseuser` module.

It adds record rules so that Employee users can create Tasks, and can edit them while in Draft or Cancelled states. Once a Task is started, only Project users are allowed to write on them. More interestingly perhaps, `project_issue_baseuser` extends that to Project Issues.

In practice, and depending on Project configuration, several Stages can map to the "Draft" state.
Keeping the record rules attached to a generic State allows Project Managers to organize their Project Stages at will, without needing to touch record rules configurations.

**Comment by pedrobaeza** [comment]
Another one that has been merged automatically... Well, your explanation convinced me, so for my part :+1: Let's see if anyone has any problem with this.

**Comment by dreispt** [comment]
What do you mean with "merged automatically"?

**Comment by dreispt** [comment]
OK I got it. I warned about it on the PRs description :smile:

**Pull request merged**